bindings/nodejs/src/lib.rs

```diff
@@ -311,6 +322,28 @@ impl Lister {
     }
 }
 
+
+#[napi(iterator)]
+pub struct BlockingLister(opendal::BlockingLister);
+
+#[napi]
+impl Generator for BlockingLister {
+    type Yield = Result<Entry>;
+    type Next = ();
```

Comment:
What's the meaning of `Next` and `Return`? Can you add some comments here to explain.

Reply:
Updated.